src/applications/claims-status/tests/e2e/02.claim-files.v2.cypress.spec.js

```diff
@@ -0,0 +1,70 @@
+import TrackClaimsPageV2 from './page-objects/TrackClaimsPageV2';
```

Comment:
Looks good! These were all on the list of things to test! Also glad you added the file upload error test.